root.tex

```diff
@@ -56,10 +55,10 @@
 
 
 \section{Introduction}
-This document provides a report of Team JSK’s progress in preparing for the Mohamed Bin Zayed International Robotics Challenge (MBZIRC). The team consists of members from the JSK Laboratory at the University of Tokyo. The JSK Lab, founded in early 1980’s, has a long history of robotics research with focus on areas including humanoids, drones, robotics manipulation, and perception. The lab has experience in participating in robotics challenges including the DARPA Robotic Challenge and the Amazon Picking Challenge.
+This document provides a report of Team JSK's progress in preparing for the Mohamed Bin Zayed International Robotics Challenge (MBZIRC). The team consists of members from the JSK Laboratory at the University of Tokyo. The JSK Lab, founded in early 1980’s, has a long history of robotics research with focus on areas including humanoids, drones, robotics manipulation, and perception, and the lab has experience in participating in robotics challenges including the DARPA Robotic Challenge and the Amazon Picking Challenge.
 
-\section{Project Personnel}
-Team JSK is made of 11 members. The 11 members are Prof. Inaba, Prof. Okada, Dr. Kakiuchi, Dr. W. Chan, Bakui Chou, Xiangyu Chen, Krishneel Chaudhary, Kohei Kimura, Yuki Furuta and Hiroto Mizohana. The team is divided into 3 groups generally to handle various tasks such as software and hardware developments and other project related activities. All team members are from JSK Lab, in The University of Tokyo.
+\subsection{Project Personnel}
+Team JSK is made of eleven members: Prof. Masayuki Inaba, Prof. Kei Okada, Dr. Yohei Kakiuchi, Dr. Wesley Chan, Bakui Chou, Xiangyu Chen, Krishneel Chaudhary, Kohei Kimura, Yuki Furuta, and Hiroto Mizohana. The team is roughly divided into 3 groups corresponding to each task to handle software and hardware develThen just remove the email address and give the JSK websiteopments and other project related activities.
```

Comment:
@wesleypchan  Please check the last sentence of this line. Its very very ambiguous.